ui/src/app/pages/data/data-set/new-dataset-modal.spec.tsx

```diff
@@ -11,7 +11,10 @@ import {
 import {waitOneTickAndUpdate} from 'testing/react-test-helpers';
 import {DataSetApiStub} from 'testing/stubs/data-set-api-stub';
 import {WorkspacesApiStub} from 'testing/stubs/workspaces-api-stub';
+import {DataSetExportRequest} from '../../../../generated/fetch';
 import {NewDataSetModal} from './new-dataset-modal';
+import GenomicsAnalysisToolEnum = DataSetExportRequest.GenomicsAnalysisToolEnum;
```

Comment:
interesting - I actually haven't seen this pattern before but it seems pretty nice